static/js/pages/instances/InstanceDetailPanel.tsx

```diff
@@ -30,107 +50,183 @@ const InstanceDetailPanel: FC<Props> = ({ instance, notify, onClose }) => {
         );
       });
```

Comment:
When we add `?? []` to the end of  `getIpAddresses`, it always returns an array. Then we can save a bunch of checks down below.